include/sicks3000.h

```diff
@@ -60,14 +60,17 @@ class SickS3000
     //! Close the port
     int Close();
 
+    //! Check the operating data block of the S3000 for system and error status
+    int CheckOperatingDataBlock(int retry_count);
+
     //! Read and process data
-    int ReadLaser( sensor_msgs::LaserScan& scan_msg, bool& bValidData ); // public periodic function
+    int ReadLaser(sensor_msgs::LaserScan& scan_msg, bool& bValidData); // public periodic function
```

Comment:
I'm not sure of the styling here, but for now I've at least made the files consistent.  Will need to roslint everything once all the review changes are done.